controller/scheduler/fleet.py

```diff
@@ -377,6 +376,7 @@ def attach(self, name):
 
 CONTAINER_TEMPLATE = [
     {"section": "Unit", "name": "Description", "value": "{name}"},
+    {"section": "Service", "name": "EnvironmentFile", "value": "/etc/environment"},
```

Comment:
App units rely on etcd to resolve registry host and port.  If etcd isn't running locally, ideally `ETCDCTL_PEERS` is set to enable `etcdctl` to use _remote_ etcds for that purpose.

The idea was that anyone wanting to run their data plane in an elastic pool of machines _not_ running etcd (which seems a semi-advanced use case) could use cloud-config to poke a value for `ETCDCTL_PEERS` into `/etc/environment`.

Since there is possibly a security concern here, what if I kept this approach, but used cloud-config to poke `ETCDCTL_PEERS` into a _different_ file instead, like `/etc/app-environment`?  It would lack anything potentially sensitive or ripe for abuse like `COREOS_PRIVATE_IPV4`.

Forthcoming documentation would illustrate how `ETCDCTL_PEERS` can be poked into that file using cloud-config.